src/main/kotlin/racingcarWinner/ui/OutputView.kt

```diff
@@ -1,38 +1,32 @@
 package racingcarWinner.ui
 
-import racingcarWinner.core.Car
-import racingcarWinner.core.MessageCode
-import racingcarWinner.core.Winner
-import racingcarWinner.core.WinnerRacing
+import racingcarWinner.domain.Car
+import racingcarWinner.util.MessageCode
 
 object OutputView {
     private const val MOVE_STEP = "-"
     private const val SEPARATOR = " "
     private const val COMMA = ", "
     private const val COLON = " : "
     private const val ESCAPE_LINE = "\n"
-    private const val ESCAPE_DOUBLE_LINE = "\n\n"
 
-    fun outputResult(cars: List<Car>, winner: Winner, tryNumber: Int) {
-        println(MessageCode.Result.message)
-        print(tryAllRacingOutput(cars, winner, tryNumber))
+    fun outputResult() {
+        println(ESCAPE_LINE.plus(MessageCode.Result.message))
```

Comment:
```suggestion
        println(MessageCode.Result.message)
```
`println`이 마지막에 엔터를 해주니 이렇게 바로 표현해주셔도 됩니다.